docs/asciidoc/modules/ROOT/pages/ml/openai.adoc

```diff
@@ -249,3 +250,53 @@ CALL apoc.ml.openai.chat([{"role": "user", "content": "Explain the importance of
     '<apiKey>',
     {endpoint: 'https://api.groq.com/openai/v1', model: 'mixtral-8x7b-32768'})
 ----
+
```

Comment:
add examples for additional parameters (max_tokens, max_tokens_sample, ...)

Reply:
Added below